src/test/scala/com/databricks/spark/avro/AvroSuite.scala

```diff
@@ -29,6 +29,15 @@ import org.scalatest.FunSuite
 /* Implicits */
 import TestSQLContext._
 
+private[avro] object DirectoryDeletion {
```

Comment:
But I might actually copy Spark's implementation of the same thing from here: https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/util/Utils.scala#L908 (remove the shutdownDeletePaths thing), with a note of where it came from and after putting it in its own utility class.